src/commands/database-get.ts

```diff
@@ -99,40 +98,51 @@ module.exports = new Command("database:get <path>")
     dbUrl = urlObj.href;
 
     logger.debug("Query URL: ", dbUrl);
-    var reqOptions = {
+    const reqOptions = {
       url: dbUrl,
     };
 
-    return api.addRequestHeaders(reqOptions).then(function(reqOptionsWithToken) {
-      return new Promise(function(resolve, reject) {
-        var fileOut = !!options.output;
-        var outStream = fileOut ? fs.createWriteStream(options.output) : process.stdout;
-        var erroring;
-        var errorResponse = "";
-        var response;
+    return api.addRequestHeaders(reqOptions).then((reqOptionsWithToken) => {
+      return new Promise((resolve, reject) => {
+        const fileOut = !!options.output;
+        const outStream = fileOut ? fs.createWriteStream(options.output) : process.stdout;
+        const writeOut = (s: Buffer | string, cb?: Function): void => {
```

Comment:
The branches below are rather involved... Have we considered this?

```suggestion
        const writeOut = (s: Buffer | string, cb?: Function = () => {}): void => {
```

(and unify the branches below)

Reply:
I went and tried, but unfortunately I still get a very annoying error:

> Each member of the union type '{ (chunk: any, cb?: ((error: Error | null | undefined) => void) | undefined): boolean; (chunk: any, encoding?: string | undefined, cb?: ((error: Error | null | undefined) => void) | undefined): boolean; } | { ...; }' has signatures, but none of those signatures are compatible with each other.

It's just a subtle difference between the two that stinks.

Fortunately - I'm working on the next version of this that doesn't use `request` and this logic _does_ go away (kinda), so it'll be nicer in the very near future.

Reply:
In that case, I don't think it's worth our time to dig into this any more. Resolving.